youtube_dl/extractor/streamsb.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import random
+import string
+
+from .common import InfoExtractor
+
+
+def to_ascii_hex(str1):
+    return ''.join([format(ord(c), 'x') for c in str1])
+
+
+def generate_random_string(length):
+    return ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(length))
+
+
+class StreamsbIE(InfoExtractor):
+    domain = 'viewsb.com'
```

Comment:
If the idea is to handle other domains whose pages have the same structure, make this 
```suggestion
    _DOMAINS = ('viewsb.com', )
```
and use `_VALID_URL = r'https://(?P<domain>%s)/(?P<id>.+)' % '|'.join(_DOMAINS)`. Then you can extract the matched domain by changing l.32.

Otherwise
```suggestion
    _DOMAIN = 'viewsb.com'
```